README_es.MD

```diff
@@ -281,7 +280,8 @@ Cada función intrínseca tiene una firma de `doIntrinsicXYZ (ref, key)` y es ll
 el valor resuelto de la función. Por ejemplo, para Fn :: Sub, una entrada de `"Mi $ {MiInstancia}"`
  devolvería una cadena similar a `"Mi i-0a0a0a0a0a`.
 
-[//]: # (Translation Section:Contributions)
-## Contribuciones
+## Translations
```

Comment:
Can you leave the contributions section at the bottom of the document? Putting translations above it.